src/Config/ObjectModel/HostOptions.cs

```diff
@@ -1,6 +1,45 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Net;
+using System.Text.Json.Serialization;
+using Azure.DataApiBuilder.Service.Exceptions;
+
 namespace Azure.DataApiBuilder.Config.ObjectModel;
 
-public record HostOptions(CorsOptions? Cors, AuthenticationOptions? Authentication, HostMode Mode = HostMode.Production);
+public record HostOptions
+{
+    /// <summary>
+    /// Dab engine can at maximum handle 187 MB of data in a single response from a source.
+    /// Json deserialization of a response into a string has a limit of 197020041 bytes which when converted to MB is 187 MB.
```

Comment:
Where does 197020041 number come from? Limit in System.Text.Json dserializer ? Can we be more specific?

Reply:
Rohan, you can link to Microsoft Learn docs and to a section of code in .NET's github repo.

Reply:
Added.